src/actions/content.js

```diff
@@ -321,15 +322,29 @@ function* saveContent({ payload: { content } }) {
   try {
     yield put(resetLoading());
     yield put(showLoading());
-    const {
-      data: {
-        attributes: { title, nid },
-        type,
+    const [
+      {
+        data: {
+          attributes: { title, nid },
+        },
       },
-    } = yield call(api, 'node:save', { parameters: { node: content } });
+    ] = yield all([
+      api('node:save', { parameters: { node: content } }),
+      put(requestContentTypes()),
```

Comment:
Your are loading content types unconditionally. Better to check if we have the content type already in the store than if not load it.